packages/app-store/zoomvideo/api/callback.ts

```diff
@@ -35,6 +35,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         create: {
           type: "zoom_video",
           key: responseBody,
+          appId: "zoom",
```

Comment:
Looks correct. But it would only fix credentials created going forward. We should run query to update existing credentials as well.